src/agat/agat_convert_mfannot2gff/script.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
```

Comment:
Please make sure that the script fails when something goes wrong (e.g. `set -eo pipefail`)

Reply:
I don't understand, can you explain to me why it would not fail in case something goes wrong with using set `-eo pipefail`? I see that most component do not use that